Add funding information to the home page (#818)
napari/docs: docs/community/team.md

You can read a full history of the project’s founding and early development in this [blog post](https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python/) from Juan.

Over time, napari has grown to over 180 direct contributors. Talley Lambert,
from Harvard Medical School, began contributing around the time of Juan's blog
post, and has been a critical driver of napari's development, including
development of its first and second (current) plugin systems.

> We have now have core team members located in four continents and three

Reviewer: ```suggestion
We now have core team members located in four continents and three
```

fortnightly community calls spread across the day to accommodate users
and contributors in any time zone.

Read more about napari’s [mission and values](mission-and-values), how to get
started as a [contributor](napari-contributing) or join us in our
[public chat room](https://napari.zulipchat.com/login/). You can also follow us
on [mastodon](https://fosstodon.org/@napari),
[Bluesky](http://bsky.app/profile/napari.org), or
[LinkedIn](https://www.linkedin.com/company/napari).

(funding)=
## Institutional and funding partners

napari is developed by a global community of scientists, with many working as
volunteers, improving napari for their own use while sharing their work with
the world. We appreciate their efforts!

We have also received generous direct funding and in-kind support that
enables us to develop faster:

- The [Chan Zuckerberg Initiative] has long supported napari in numerous ways
  (listed in reverse chronological order):

>   - CZI is currently supporting napari with [two grants][sustainability]

Reviewer 2: @jni I know you don't like the name "scaffolding grant" but I really think it should be a separate line item that also lists the amount (since we don't have a grant page for that one)

Author: Done! Please check the wording. 🙏 And modify as you please...

You can read a full history of the project’s founding and early development in this [blog post](https://ilovesymposia.com/2019/10/24/introducing-napari-a-fast-n-dimensional-image-viewer-in-python/) from Juan.

Over time, napari has grown to over 180 direct contributors. Talley Lambert,
from Harvard Medical School, began contributing around the time of Juan's blog

> post, and has been a critical driver of napari's development, including

Reviewer 3: ```suggestion
post, and was a critical driver of napari's maturation, including
```
Maybe? The current grammatical structure still indicates continued involvement by Talley.

Author: See above:

> This PR does not do:
> - bring the project history up to date

I think that warrants further discussion, so I'd like to leave such changes for later PRs.

I am hesitant with this change, for example, because Talley still maintains pyapp-kit, which we heavily rely on, so he is indirectly supporting the project.
